### ISSUE TITLE
Adding uncertainties to Figure10

### ISSUE THREAD

**Issue by mostafa-razavi** [mostafa-razavi/ITIC-paper#35]
@ramess101 @jrelliottoh 

In Siepmann's TraPPE papers he reports the uncertainties as subscripts. Specifically, he says "Subscripts show the statistical uncertainty of the final digit". It's a little confusing because sometimes the subscript has two digits itself. For example, how would you interpret 1039_29 (read 1039 sub 29)?  Is this equivalent to 1039 +/- 29 or is it 1039 +/- 2.9? How about  0.000023_11? Is it 0.000023 +/- 0.0000011 or 0.000023 +/- 0.000011?

**Comment by ramess101** [comment]
@mostafa-razavi @jrelliottoh 

Here is how you should interpret those values:

1039_29 would be 1039 +- 29
0.000023_11 would be 0.000023 +- 0.000011

In other words, the uncertainty should never have a digit beyond the final digit in the reported value

**Comment by ramess101** [comment]
@mostafa-razavi 

What is really challenging to know is what type of uncertainty they are reporting. In their early studies, Siepmann always reported standard deviations or standard errors. But recently I think they report 95% confidence intervals.

**Comment by mostafa-razavi** [comment]
@ramess101 
> In other words, the uncertainty should never have a digit beyond the final digit in the reported value

I still have some doubts. What you are saying makes sense and I've been interpreting the same way so far. However, when I plot the error bars for vapor density data of TraPPE-iC4 at T=168,198, and 258 K using the first method (e.g. 0.000023_11 = 0.000023 +/- 0.0000011) I get a much better consistency with the neighboring points' error bars. This is also true for liquid density data at T=168 and 228 K.

Here is the iC4 data from TraPPE website. (The subscript for vapor density points at T=168,198, and 258 K has two digits):
![image](https://user-images.githubusercontent.com/16358113/52376740-e200a280-2a30-11e9-9644-c2396d83308c.png)

Error bars using the first method (e.g. 0.000023_11 = 0.000023 +/- 0.0000011). Look at the TraPPE-iC4 in rho_vap and rho_liq plots:
![deviation-psat-rhol-rhov](https://user-images.githubusercontent.com/16358113/52376663-a960c900-2a30-11e9-93a2-e86da5a8a400.png)

Error bars using the second method (e.g. 0.000023_11 = 0.000023 +/- 0.000011) 
![deviation-psat-rhol-rhov](https://user-images.githubusercontent.com/16358113/52376555-6999e180-2a30-11e9-9854-e2d2e1e0dd01.png)

Since the paper also specifically mentions "Subscripts show the statistical uncertainty of the **final** digit", I think the first method is what they meant. Don't you think?

**Comment by ramess101** [comment]
I am pretty confident it is the second method. Otherwise it would make no
sense to report “10” for Psat instead of just “1.”

But I do see your point about consistent error bars.

Ive looked into this before for the TraPPe website values, but maybe this
study did something weird.

**Comment by ramess101** [comment]
In the end, are we even going to include all the error bars in this plot?
At most we were just going to include a few representative error bars,
right? Or we could include a figure with all the error bars just in the SI?

**Comment by mostafa-razavi** [comment]
>In the end, are we even going to include all the error bars in this plot? At most we were just going to include a few representative error bars, right? Or we could include a figure with all the error bars just in the SI?

![deviation-psat-rhol-rhov](https://user-images.githubusercontent.com/16358113/52380729-f3e84280-2a3c-11e9-91a6-5f31babd9fd6.png)

Here is what I have so far (I also updated TraPPE-C2 results based on Tr min=0.3). I think this figure is not all that complicated. But the ITIC uncertainty assumes zero uncertainty in temperature which is not true. In addition, your concern about types of uncertainties being different is valid. So maybe we should just put this in SI.

**Comment by mostafa-razavi** [comment]
> If the error bars can't be added to figure 10, then some description of how big they would still be helpful.  Are they smaller than the symbol size (it looks like they might be in many cases)?  Where they disagree with GEMC/GCMC, is it due to uncertainty or systematic
deviation?

Alternatively, we could just do what the reviewer is really asking, i.e. explain in words how error bars would look like.

**Comment by ramess101** [comment]
@mostafa-razavi 

> I think this figure is not all that complicated

Yeah, it looks good, but this doesn't have all the uncertainties included yet, does it?

> But the ITIC uncertainty assumes zero uncertainty in temperature which is not true.

True, but would you expect the Tsat uncertainties to be larger than the symbol width? I think we talked about this once before and you said that your plotting package won't let you do errors in x-axis, is that correct?

**Comment by ramess101** [comment]
> Alternatively, we could just do what the reviewer is really asking, i.e. explain in words how error bars would look like.

Well the reviewers only said to use words "If the error bars can't be added to Figure 10." So if the error bars really don't clutter everything, I think we can just include them all. But what does the figure look like with all the error bars (even ITIC)?

**Comment by mostafa-razavi** [comment]
> Well the reviewers only said to use words "If the error bars can't be added to Figure 10." So if the error bars really don't clutter everything, I think we can just include them all. But what does the figure look like with all the error bars (even ITIC)?

ITIC results do have error bars in the above figure. They are just small in some cases and hard to see. 

> True, but would you expect the Tsat uncertainties to be larger than the symbol width? I think we talked about this once before and you said that your plotting package won't let you do errors in x-axis, is that correct?

My plotting packages (gnuplot) can handle the x-axis error bars. The problem with x-axis error bars for ITIC results in Figure 10 is that Tsat is changing so REFPROP values will change too. When T does not have uncertainty you have:

Dev % = (P_sim - P_refprop)/P_refprop X 100
Upper value for error bar = ( (P_sim+P_std) - P_refprop)/P_refprop X 100
Lower value for error bar = ( (P_sim-P_std) - P_refprop)/P_refprop X 100

If P_sim=1.5 +- 0.1 MPa at T=200 +/- 0.0 K the above equation works just fine, but imagine you have P_sim=1.5 +-0.1 MPa at 200 +/- 5.0 K. Now you don't have one value for P_refprop. What would your error bars look like?

**Comment by ramess101** [comment]
OK, I see your point. But I think it would be sufficient to just have the
error bars represent uncertainty in Tsat at a constant percent deviation.
The uncertainties in Tsat arent more than a couple K, right?

**Comment by ramess101** [comment]
@mostafa-razavi 

> , your concern about types of uncertainties being different is valid. So maybe we should just put this in SI

The error bars you plotted are using the exact uncertainties reported by the author, correct? They appear to be on a reasonable scale compared to the ITIC uncertainties, so I think we are OK just stating that uncertainties were taken from the literature. I added a comment for the caption in the PDF.

**Comment by mostafa-razavi** [comment]
> The error bars you plotted are using the exact uncertainties reported by the author, correct? They appear to be on a reasonable scale compared to the ITIC uncertainties, so I think we are OK just stating that uncertainties were taken from the literature. I added a comment for the caption in the PDF.

I'm still not sure about the TraPPE isobutane rhoL error bars for the 4 highest temperatures. Do you believe those? Compare rhoL error bars for T=368 with 343 K. Why does 368 K has such large error bars?

**Comment by ramess101** [comment]
@mostafa-razavi 

The uncertainties in rhol should increase exponentially with respect to reduced temperature. The critical temperature is 408 K, which means these two data points are at Tr of 0.84 and 0.9. That is a pretty significant difference, such that I would not be surprised by the marked increase in uncertainty. Also, they only report a single uncertainty digit, so it could be that 343 is like 0.00044 and 368 is like 0.0015. So I would trust those two uncertainties.

Also, I just realized that these are from the TraPPE validation data. So it could be that whoever redid these calculations did not correctly follow the "final digit" format. We have no manuscript to compare it with, so I think you are justified in changing the uncertainties such that they follow a more appropriate trend (e.g., 0.0013 uncertainty changed to 0.00013).

Side note: because the isobutane data are from the TraPPE validation set, we need to make a clear distinction in our references (see the TraPPE website), e.g., 

B.L. Eggimann, P. Bai, A.P. Bliss, C. Bunner, Q.P. Chen, R.F. DeJaco, E. Fetisov, D.B. Harwood, T. Josephson, R.K. Lindsey, M.S. Shah, H.D. Stern, K.N. Struksheats, J. Sung, A.J. Sunnarborg, B. Xue, and J.I. Siepmann
T-UA No. 16 2-methylpropane
TraPPE Validation Database University of Minnesota: Minneapolis, MN
http://chem-siepmann.oit.umn.edu/siepmann/trappe/validation.html
(accessed 2019 February 11).

**Comment by ramess101** [comment]
@mostafa-razavi 

And perhaps more importantly, the TraPPE validation data use a target uncertainty that is different above Tr of 0.9:

![image](https://user-images.githubusercontent.com/23408516/52595815-0206dc00-2e0c-11e9-8cf0-d5588bcc2680.png)

So this would explain why higher T data could have a different relative uncertainty
